### PR TITLE
add my script -HNG00221

### PR DESCRIPTION
**Pull request by greazleay**
Script rewritten and the recommended amendments made

### CODE REVIEW

**Comment by dro-1** [comment]
Wrong Output.

**Review by alexismajya — COMMENTED** [review]
sound good!

**Comment by greazleay** [comment]
@dro-1 could you please share the expected output?

I tested locally before pushing and this was the output: "**Hello World, this is Lekan Adetunmbi with HNG ID HNG-00221 and email lothbroch@gmail.com using javascript for stage 2 task**"

File: scripts/elysium.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+let fullName = "Lekan Adetunmbi";
```
Comment (BenjaminKyamanywa) [inline scripts/elysium.js:3]: hey, @FElysian you can change 'fullName' variable to 'name'

Comment (BenjaminKyamanywa) [inline scripts/elysium.js:3]: hey, @FElysian you can change 'fullName' variable to 'name' and don't forget to update the same on your pass variable for the output.

File: scripts/elysium.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+let ID = "HNG-00221";
```
Comment (BenjaminKyamanywa) [inline scripts/elysium.js:4]: change 'ID' to 'id' for this variable to and update the same for your output pass variable. rest all looks good.

File: scripts/elysium.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+let email = "lothbroch@gmail.com";
+let language = "javascript";
```
Comment (BenjaminKyamanywa) [inline scripts/elysium.js:6]: you can also change 'javascript' to 'Javascript' with capital 'J'

**Comment by Rubylena** [comment]
what is happening, will close this. resolve your issues soon

**Pull request merged**